### ISSUE TITLE
beacnount-dkb broken due to format change: delimiter changed to semi-colon

### ISSUE THREAD

**Issue by mh182** [siddhantgoel/beancount-dkb#195]
It seems that DKB changed the format and uses a semi-colon `;` as a delimiter. All commands of `beancount-dkb-ec` return with exit code `0` silently.

```
✦ ❯ beancount-dkb-ec identify incoming/03-11-2024_Umsatzliste_Girokonto_DE1234567890.csv
* /home/max/accounting/incoming/03-11-2024_Umsatzliste_Girokonto_DE1234567890csv
* ✦ ➜ echo $?
0
```

Combined with #190 I currently have to do the following preprocessing of files to get `extract` working:
- Change the comma separator of `Saldo`  from `,` to `.`
- Replace `;` delimiters with `,`

**Comment by siddhantgoel** [comment]
I just published v1.3.0 which should support both `,` and `;` as delimiters.

**Comment by mh182** [comment]
Works now.
Thanks for your effort.
